libeos-payg/tests/manager.c

```diff
@@ -606,6 +617,7 @@ test_manager_error_rate_limit (Fixture *fixture,
   manager_new (fixture);
 
   g_autoptr(GError) error = NULL;
+  gint64 *time_added = NULL;
```

Comment:
Surely this (and the others in this file) should be `gint64 time_added = 0` not `gint64 *`?

Reply:
Ah, you are right, just read a bit about it and it does not seem necessary here.